apps/mealplanr-api/src/app.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
What does this comment do? And why is it used here?

Reply:
hm, i hid some warnings, in the panel, that i did not want to mess with - i did not know that it made comments - i'll fix them with the code smells